docs/reference/connector/docs/connectors-release-notes.asciidoc

```diff
@@ -9,6 +9,18 @@
 Prior to version *8.16.0*, the connector release notes were published as part of the {enterprise-search-ref}/changelog.html[Enterprise Search documentation].
 ====
 
+[[es-connectors-release-notes-8-18-0]]
```

Comment:
@charlotte-hoblik the lack of `[discrete]` is the culprit for this page being standalone